hidimstat/ada_svr.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
I think I'd rather not have ada_svr as a module directly under hidimstat. Should be in something like `hidimstat.estimators`.

Reply:
It's fine for me. 
Do you prefer call this module: procedures or methods or models ?

Reply:
maybe models.
But we should first discuss the organization of the whole thing to make a good decision.

Reply:
See the issue #86